client/client_test.go

```diff
@@ -0,0 +1,104 @@
+package client
```

Comment:
may it is better to have something like this in https://github.com/OpenVPN/cloudconnexa-go-client